test/fail_compilation/ice13024.d

```diff
@@ -12,5 +12,6 @@ void main()
 {
     T t;
     auto r1 = [cast(int)(t.x), cast(int)(t.y)]; // OK
-    auto r3 = [t.x, t.y]; // crash
+    // Used to be [t.x, t.y]. Behavior changed with fix of issue 20938
```

Comment:
What I meant was to change L9 from `enum B { b }` to `enum B : string { b = "42" }`

Reply:
But then L14 wouldn't compile, and much bigger changes would need to be done to the whole test. I'm starting to think `static assert(false, "This test no longer fails to compile after issue 20938 was fixed.");` might be the best solution.

The root problem is the test for 13024 checks more than it should - it should only check if `[t.x, t.y]` crashes the compiler, not that it fails with a specific error message. I don't know how possible that would be, though.

Reply:
Right, I missed that.
Then I think the best course of action here is to add to `b20938.d` a test with two enums that just checks it compiles, and remove `ice13024.d`.

Reply:
Done.